wrappers/otp.rb

```diff
@@ -63,12 +63,17 @@ def route(locs, dimension, departure, arrival, language, with_geometry, options
           wheelchair: false,
           showIntermediateStops: false
         }
-        request = RestClient.get(@url + '/otp/routers/' + @router_id + '/plan', {
-          accept: :json,
-          params: params
-        })
-        data = JSON.parse(request)
-        @cache.write(key, data) if data && !data['error'] && data['plan'] && data['plan']['itineraries']
+
+        begin
+          request = RestClient.get(@url + '/otp/routers/' + @router_id + '/plan', {
+            accept: :json,
+            params: params
+          })
+          data = JSON.parse(request)
+          @cache.write(key, data) if data && !data['error'] && data['plan'] && data['plan']['itineraries']
+        rescue RestClient::InternalServerError => ex
+          raise RouterWrapper::NoRouteFound.new(ex), "OTP, #{ex.message} on #{@router_id}"
```

Comment:
I suggest you: 
```
rescue Restclient::ExceptionWithResponse => e
  e.message += " - OTP #{@router_id}"
  raise e
```
not to overload original exception and message

Reply:
OK. done.